src/types.d.ts

```diff
@@ -8,7 +8,7 @@ export interface ISNSAdapter {
     listSubscriptions(): Promise<ListSubscriptionsResponse>;
     unsubscribe(arn: string): Promise<void>;
     createTopic(topicName: string): Promise<CreateTopicResponse>;
-    subscribe(fnName: string, handler: SLSHandler, arn: string): Promise<void>;
+    subscribe(fnName: string, handler: SLSHandler, arn: string, policies: any): Promise<void>;
```

Comment:
Didn't see this signature, should change as follows:
```suggestion
    subscribe(fnName: string, handler: SLSHandler, arn: string, snsConfig: any): Promise<void>;
```